tests/routes/test_join.py

```diff
@@ -130,6 +131,63 @@ def test_join_selection_other_user(self):
         self.assertIn("Office 365", str(response.data))
 
 
+class TestSendInvitations(unittest.TestCase):
+    def setUp(self):
+        self.github_service = MagicMock(GithubService)
+        self.app = create_app(self.github_service, False)
+        self.app.config["SECRET_KEY"] = "test_flask"
+        self.client = self.app.test_client()
+
+    @patch("app.main.routes.join.app_config", new_callable=MagicMock)
+    @patch('app.main.routes.join.is_pre_approved_email_domain', return_value=True)
+    def test_when_user_is_already_member_of_an_org(self, mock_app_config, mock_is_pre_approved_email_domain):
+
+        mock_app_config.return_value = SimpleNamespace(
+            github=SimpleNamespace(
+                send_email_invites_is_enabled=True,
+                token="mock_token",
+                allowed_email_domains=[
+                    "justice.gov.uk"
+                ],
+                organisations=[
+                    SimpleNamespace(
+                        name="ministryofjustice",
+                        enabled=True,
+                        display_test="Mock"
+                    )
+                ]
+            )
+        )
+
+        self.github_service.send_invites_to_user_email.side_effect = GithubException(
+            status=422,
+            data={
+                "message": "Validation Failed",
+                "errors": [
+                    {
+                        "resource": "OrganizationInvitation",
+                        "code": "unprocessable",
+                        "field": "data",
+                        "message": "A user with this email address is already a part of this organization"
+                    }
+                ],
+                "documentation_url": "https://docs.github.com/rest/orgs/members#create-an-organization-invitation"
+            }
+        )
+
+        with self.client.session_transaction() as sess:
+            sess["user"] = {"userinfo": {"email": "test@justice.gov.uk"}}
+            sess["user_input_email"] = "test@justice.gov.uk"
+            sess["org_selection"] = ["ministryofjustice"]
+
+        response = self.client.get("/join/send-invitation")
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.request.path, "/join/send-invitation")
+        self.assertIn(
+            "It appears you are already a member", str(
+                response.data))
+
+
```

Comment:
Could we move the tests from the Class below into this test Class? 👇 👀 

Just feels easier to read if all the tests for one function are under the same class 🤗 But happy for that to be a separate PR 😁